os/include/tinyara/lwnl/lwnl.h

```diff
@@ -77,6 +79,11 @@ typedef struct {
 	void *result;
 } lwnl_msg;
 
+typedef struct {
```

Comment:
this structure is only for ble.  so how about to move it to another header file. for example tinyara/net/if/ble.h.

Reply:
I think this can also be used in other device when "lwnl" is using.
There is no way to deliver more than 2 params in lwnl